### PR TITLE
Add honeybadger docs

### PR DESCRIPTION
**Pull request by SaravShah**
When we are setting up a new rails project, we should use Honeybadger for application monitoring. Otherwise, we can't keep track of an applications exceptions or deployments as easily.

paired w/ @tallenaz

### CODE REVIEW

File: best-practices/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+If you decide on the former approach, you will want to set up your honeybadger config file in [sul-dlss/shared_configs](https://github.com/sul-dlss/shared_configs). If you would like to use environment variables instead, you will need some assistance from operations to encrypt your project's API key and set up the necessary environment variables via [sul-dlss/puppet](https://github.com/sul-dlss/puppet).
+
+### Slack Integrations
+
+- In the Slack application go to the side bar under `Direct Messages` and click on the Honeybadger application. This should pop up a right side bar where you can click Settings. Then click on "Add Configuration". Then follow the instructions given.
+- Set up a configuration with your PROD environment to the deploy channel using this webhook URL: `https://stanfordlib.slack.com/services/hooks/honeybadger?token=oT20rCGAYNQ7K0QQoWy0IIO4`.
```
Comment (mejackreed) [inline best-practices/monitoring.md]: Is this token something that we should commit here in a public repo? I'm not sure of the implications here.

Comment (SaravShah) [reply best-practices/monitoring.md]: Updated - but how do we remove it from this Github view now?

Comment (camillevilla) [reply best-practices/monitoring.md]: @SaravShah Gotta contact GitHub support

File: best-practices/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+The [honeybadger docs](https://docs.honeybadger.io/ruby/integration-guides/rails-exception-tracking.html) contain instructions for how to set it up in a Rails application, but there is a wrinkle to consider as you do so. The instructions point out that installing will "[g]enerate a config/honeybadger.yml file. If you don't like config files, you can place your API key in the $HONEYBADGER_API_KEY environment variable."
+
+If you decide on the former approach, you will want to set up your honeybadger config file in [sul-dlss/shared_configs](https://github.com/sul-dlss/shared_configs). If you would like to use environment variables instead, you will need some assistance from operations to encrypt your project's API key and set up the necessary environment variables via [sul-dlss/puppet](https://github.com/sul-dlss/puppet).
+
+### Slack Integrations
+
+- In the Slack application go to the side bar under `Direct Messages` and click on the Honeybadger application. This should pop up a right side bar where you can click Settings. Then click on "Add Configuration". Then follow the instructions given.
+- Set up a configuration with your PROD environment to the deploy channel.
```
Comment (camillevilla) [inline best-practices/monitoring.md]: Just submitted a PR (#75)  that walks through this part of the process. Hoping these both get approved and can get linked up 🖇

**Pull request merged**